Lib/test/test_shutil.py

```diff
@@ -727,6 +727,17 @@ def test_copytree_return_value(self):
         rv = shutil.copytree(src_dir, dst_dir)
         self.assertEqual(['foo'], os.listdir(rv))
 
+    def test_copytree_subdirectory(self):
+        # copytree where dst is a subdirectory of src, see Issue 38688
+        base_dir = self.mkdtemp()
+        src_dir = base_dir + "t/pg/"
+        dst_dir = src_dir + "somevendor/1.0"
```

Comment:
use `os.path.join()` instead

Reply:
Almost missed this one. Just a few lines below in this same PR there is already an `os.path.join`, no reason to not use it here too, good spot again! Thanks!